docs/build-provenance-for-all-package-registries.md

```diff
@@ -32,7 +32,7 @@ These requirements mean you won't be able to generate trustworthy build provenan
 
 Instead, we recommend that **builds take place on cloud CI/CD systems** that are able to provide isolation, ephemeral runners, and signed build metadata that the build cannot tamper with. The information from these cloud CI/CD systems are sent to Sigstore's Fulcio component. You can see what cloud CI/CD systems Fulcio trusts at <https://github.com/sigstore/fulcio/tree/main/federation>, and you can see what information those cloud CI/CD systems must provide at <https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md#requirements-to-support-signing-with-cicd-workload-identities>.
 
-Note that **build provenance does not try to tie a package back to a verified human identity**. Instead we're focusing on the workload identity from the build system, as that is what produces the package. Strongly authenticating humans and trying to determine their intent is a complex problem space that is out of scope for build provenance!
+Note that **build provenance does not try to tie a package back to a verified human identity**. Instead we're focusing on the workload identity from the build system, as that is what produces the package. Strongly authenticating humans and trying to determine their intent is a complex problem space that is out of scope for build provenance! Further, authenticating humans raises privacy and legal concerns that pose obstacles to repository adoption for signatures; prioritizing build provenance sidesteps these.
```

Comment:
```suggestion
Note that **build provenance does not try to tie a package back to a verified human identity**. Instead we're focusing on the workload identity from the build system, as that is what produces the package. Strongly authenticating humans and trying to determine their intent is a complex problem space that is out of scope for build provenance! Further, authenticating humans often involves Personally Identifiable Information, which registries would like to avoid additional handling of for privacy and legal concerns. By using workload identity for build provenance, registries can avoid these issues.
```